src/main/java/seedu/address/logic/commands/EditMeetingCommand.java

```diff
@@ -0,0 +1,186 @@
+        return new CommandResult(String.format(MESSAGE_EDIT_MEETING_SUCCESS, editedMeeting));
+    }
+
+    /**
+     * Creates and returns a {@code Meeting} with the details of {@code meetingToEdit}
+     * edited with {@code editMeetingDescriptor}.
+     */
+    private static Meeting createEditedMeeting(ReadOnlyMeeting meetingToEdit,
+                                              EditMeetingDescriptor editMeetingDescriptor) {
+        assert meetingToEdit != null;
+
+        NameMeeting updatedName = editMeetingDescriptor.getName().orElse(meetingToEdit.getName());
+        DateTime updatedDate = editMeetingDescriptor.getDate().orElse(meetingToEdit.getDate());
+        Place updatedPlace = editMeetingDescriptor.getPlace().orElse(meetingToEdit.getPlace());
+        //Set<Tag> updatedTags = editMeetingDescriptor.getTags().orElse(meetingToEdit.getTags());
+
+        return new Meeting(updatedName, updatedDate, updatedPlace);
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        // short circuit if same object
+        if (other == this) {
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof EditMeetingCommand)) {
+            return false;
+        }
+
+        // state check
+        EditMeetingCommand e = (EditMeetingCommand) other;
+        return index.equals(e.index)
+                && editMeetingDescriptor.equals(e.editMeetingDescriptor);
+    }
+
+    /**
+     * Stores the details to edit the Meeting with. Each non-empty field value will replace the
+     * corresponding field value of the meeting.
+     */
+    public static class EditMeetingDescriptor {
+        private NameMeeting name;
+        private DateTime date;
+        private Place place;
+
+        public EditMeetingDescriptor() {
+        }
+
+        public EditMeetingDescriptor(EditMeetingCommand.EditMeetingDescriptor toCopy) {
+            this.name = toCopy.name;
+            this.date = toCopy.date;
+            this.place = toCopy.place;
+        }
+
+        /**
+         * Returns true if at least one field is edited.
+         */
+        public boolean isAnyFieldEdited() {
+            return CollectionUtil.isAnyNonNull(this.name, this.date, this.place);
+        }
+        public void setNameMeeting(NameMeeting name) {
+            this.name = name;
+        }
+
+        public Optional<NameMeeting> getName() {
+            return Optional.ofNullable(name);
+        }
+
+        public void setDate(DateTime date) {
+            this.date = date;
+        }
+
+        public Optional<DateTime> getDate() {
+            return Optional.ofNullable(date);
+        }
+
+        public void setPlace(Place place) {
+            this.place = place;
+        }
+
+        public Optional<Place> getPlace() {
+            return Optional.ofNullable(place);
+        }
+
+        @Override
+        public boolean equals(Object other) {
```

Comment:
Just to be consistent, may be you can follow the code style in other equals method, such as:

```
return other == this // short circuit if same object
                || (other instanceof ReadOnlyMeeting // instanceof handles nulls
                && this.isSameStateAs((ReadOnlyMeeting) other));
```